app/jobs/health/process_enrollment_changes_job.rb

```diff
@@ -0,0 +1,88 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module Health
```

Comment:
This is fine for now, but if these files become large, it will probably make sense to batch the process such that it loads all of the medicaid_ids and action (enroll/disenroll) from the enrollment file and then load the patients in batches.  Finally, instead of calling `update` on each patient, possibly use `active_record_import` to do an upsert.